packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -3292,13 +3292,8 @@ class RenderEditable extends RenderBox with RelayoutWhenSystemFontsChangeMixin,
       final bool isHit = result.addWithPaintTransform(
         transform: transform,
         position: position,
-        hitTest: (BoxHitTestResult result, Offset? transformed) {
-          assert(() {
-            final Offset manualPosition = (position - textParentData.offset) / textParentData.scale!;
-            return (transformed!.dx - manualPosition.dx).abs() < precisionErrorTolerance
-              && (transformed.dy - manualPosition.dy).abs() < precisionErrorTolerance;
-          }());
```

Comment:
This is the same as other asserts, but with more shenanigans around precision error tolerance and transform